src/plugins/plot/src/configuration/PlotSeries.js

```diff
@@ -322,7 +322,15 @@ define([
          *                  a point to the end without dupe checking.
          */
         add: function (point, appendOnly) {
-            var insertIndex = this.data.length;
+            var insertIndex = this.data.length,
+                currentYVal = this.getYVal(point),
+                lastYVal = this.getYVal(this.data[insertIndex - 1]);
+
+            if (this.isValueInvalid(currentYVal) && this.isValueInvalid(lastYVal)) {
+                console.warn('[Plot] Invalid Y Values detected');
```

Comment:
May be use openmct notifications or remove?

Reply:
I didnt want to bombard the user, because they could be 100's of potential invalid values

Reply:
Still need a message, so i decided to go via console